doc/conf.py

```diff
@@ -167,7 +167,8 @@
 #modindex_common_prefix = []
 
 nitpicky = True
-nitpick_ignore = [
+nitpick_ignore = [ ('cpp:typeOrConcept', 'T')
+                 , ('cpp:typeOrConcept', 'Args')
```

Comment:
Seems to build fine with Sphinx==1.7.5 (our minimum required version) and with Sphinx==1.7.6 (latest). We can do without these exceptions.

Reply:
I needed this for Sphinx 1.7.6 or I get warnings for these values.
```
writing output... [100%] index                                                                                                           
/home/amc/git/ats/doc/developer-guide/internal-libraries/MemArena.en.rst:127: WARNING: cpp:identifier reference target not found: T
/home/amc/git/ats/doc/developer-guide/internal-libraries/MemArena.en.rst:127: WARNING: cpp:identifier reference target not found: Args
```

Reply:
It's phase of the moon, but after several runs without errors, I got this
```
Running Sphinx v1.7.6
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 1 source files that are out of date
updating environment: 0 added, 1 changed, 0 removed
reading sources... [100%] developer-guide/internal-libraries/MemArena.en                                                                 
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] index                                                                                                           
/home/amc/git/ats/doc/developer-guide/internal-libraries/MemArena.en.rst:123: WARNING: cpp:identifier reference target not found: T
/home/amc/git/ats/doc/developer-guide/internal-libraries/MemArena.en.rst:123: WARNING: cpp:identifier reference target not found: Args
generating indices... genindex
writing additional pages... search
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded, 2 warnings.
```

Reply:
I need these in here in order to consistently get clean runs without warnings.

Reply:
Weird. I've still not seen this in my own environment. I agree if this is brittle, better to have the exceptions.